ui/admin/app/templates/scopes/scope/targets/target/manage-alias.hbs

```diff
@@ -3,6 +3,11 @@
   SPDX-License-Identifier: MPL-2.0
 }}
 
+<Breadcrumbs::Item
```

Comment:
Why do we have a Breadcrumbs item outside of the layout?

Reply:
This component adds the current route to the internal array that keeps track of the items which is why it's outside the layout. The Breadcrumbs::Container is what renders the actual breadcrumbs items list.